generate.sh

```diff
@@ -15,6 +15,10 @@ do
  esac
 done
 
+function print_log {
+  echo -e $1 | tee -a ${LOGFILE}
```

Comment:
Fallback to simple echo when `LOGFILE` is not defined. Thus it wouldn't break on travis